### ISSUE TITLE
Suggestion: Do not commit node_modules

### ISSUE THREAD

**Issue by melissamcewen** [ProgrammingJoe/Codecademy-TDD-GatherSubmission#2]
I noticed this took a bit of time for me to download and then I realized it included node_modules. There are several reasons you might not want to include this. One is that it takes up a lot of room but here are some more
https://byjoeybaker.com/why-you-should-never-commit-node-modules-in-nodejs
To omit it, I use .gitignore
https://chaseadams.io/2015/07/my-gitignore-conventions/

**Comment by ProgrammingJoe** [comment]
Yeah, I've actually re-uploaded this project to tidy up my github and in that version I remove and ignore node_modules
